chainerx_cc/CMakeLists.txt

```diff
@@ -51,6 +51,9 @@ endif()
 
 # CUDA
 if(${CHAINERX_BUILD_CUDA})
+    option(CUDA_USE_STATIC_CUDA_RUNTIME "Use the static version of the CUDA runtime library if available" OFF)
```

Comment:
Does this change the default behavior for all platforms?

Reply:
fixed it
https://github.com/durswd/chainer/commit/4cf5c76dcfa773948edababfefaac709b5634711

Reply:
Thanks.